oap-server/server-bootstrap/src/main/resources/ui-initialized-templates/self-observability.yml

```diff
@@ -184,6 +184,39 @@ templates:
                   "metricLabels": "50,70,90,99",
                   "labelsIndex": "50,70,90,99",
                   "unit": "Millisecond"
+                },
+                {
+                  "width": 3,
+                  "title": "Thread Count",
+                  "height": "200",
+                  "entityType": "ServiceInstance",
+                  "independentSelector": false,
+                  "metricType": "REGULAR_VALUE",
+                  "metricName": "meter_oap_instance_jvm_thread_live_count,meter_oap_instance_jvm_thread_daemon_count,meter_oap_instance_jvm_thread_peak_count",
+                  "queryMetricType": "readMetricsValues",
+                  "chartType": "ChartLine"
+                },
+                {
+                  "width": 3,
+                  "title": "Thread State Count",
+                  "height": "200",
+                  "entityType": "ServiceInstance",
+                  "independentSelector": false,
+                  "metricType": "REGULAR_VALUE",
+                  "metricName": "meter_oap_instance_jvm_thread_runnable_state_thread_count,meter_oap_instance_jvm_thread_blocked_state_thread_count,meter_oap_instance_jvm_thread_waiting_state_thread_count,meter_oap_instance_jvm_thread_timed_waiting_state_thread_count",
```

Comment:
Could you run test on real PostgreSQL? I am feeling whether this name, `meter_oap_instance_jvm_thread_runnable_state_thread_count`, is too long for PostgreSQL.